ui.tests/test-module/libs/commons/formsConstants.js

```diff
@@ -23,7 +23,8 @@ var formsConstants = {
                 "fplinkcomponent"  : "/apps/forms-components-examples/components/linkcomponent",
                 "fpsnlcomponent"  : "/apps/forms-components-examples/components/searchlister",
                 "fpdnscomponent"  : "/apps/forms-components-examples/components/draftsandsubmissions",
-                "formtextinput": "/apps/forms-components-examples/components/form/textinput"
+                "formtextinput": "/apps/forms-components-examples/components/form/textinput",
+                "panelContainer": "/apps/forms-components-examples/components/form/panelContainer"
```

Comment:
we are not using sentence case in any of the RT paths. 
as it forms a part of URL, I wont suggest sentence case. 
it should be `panelcontainer`